### ISSUE TITLE
Doesn't allow to use bootnodes for custom chain

### ISSUE THREAD

**Issue by splix** [ethereumproject/go-ethereum#331]
```
$ geth --chain ecip1017 --bootnodes enode://5c5f016f174d8600cf1f8b3fab79eb2f8b10a58c9abab8202f6d785e09eca4a10150951674bbc0837d927cfe49c3d99d83a3efd21a55929b3ad0cfdbcf029da0@pod-G2:30303,enode://1ba61fcfeef3253973eb6379d333865c1454b3c791f631c95d67712338de94f1b161bd8e6a0690b0b910d7334f2be2d94e8524b5a2e18404b051301af277d3eb@pod-GM1:30303
Conflicting custom chain config and --bootnodes flags. Please use either but not both.
```

Why it cannot be used together?

**Comment by whilei** [comment]
It would need to be decided and specified that using `--bootnodes` would append-if-unique OR replace. Without that specification it could have unexpected results.

**Comment by splix** [comment]
replace I think. with replace you can reach both goals, just by merging existing into new list and providing it as option. also, there should be a way to remove particular bootnode, and replace works for that too

**Comment by whilei** [comment]
Agreed. While we're at it I guess we should also allow `--network-id` flag to replace file-configured `"network": 1,`

**Comment by whilei** [comment]
Resolved with #333
